Dockerfile

```
@@ -1,43 +1,46 @@
-FROM ubuntu:xenial-20181113
+FROM ubuntu:22.04
```

To avoid Docker Hub rate limiting, we should use the images mirrored at `quay.io/bedrock`:

```suggestion
FROM quay.io/bedrock/ubuntu:22.04
```

---

FYI - The bedrock mirror is managed by https://github.com/ansible/bedrock

---

ah nice.